core/tests/protractor_utils/BlogDashboardPage.js

```diff
@@ -160,7 +160,7 @@ var BlogDashboardPage = function() {
     await action.sendKeys(
       'New blog post title field', blogPostTitleFieldElement, blogPostTitle);
     await this.setContent(richTextInstructions);
-    await waitFor.visibilityOf(
+    await waitFor.presenceOf(
```

Comment:
Using `visibilityOf` resulted in e2e failures https://github.com/oppia/oppia/runs/3742440666